From 0f3212f65b6672b58feaed98d2fecca01f4c25ab Mon Sep 17 00:00:00 2001
From: Christoph Korn <c_korn@gmx.de>
Date: Sun, 15 Sep 2013 15:34:09 +0200
Subject: [PATCH] Makefile: Add geoip so it gets installed

Without adding it to CORE it does not get installed and the game crashes.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index e23ac09..41078e5 100644
--- a/Makefile
+++ b/Makefile
@@ -54,7 +54,7 @@ INSTALL_PROGRAM = $(INSTALL) -m755
 INSTALL_DATA = $(INSTALL) -m644
 
 # program targets
-CORE = fileformats rcg rgl rsdl rnull game utility
+CORE = fileformats rcg rgl rsdl rnull game utility geoip
 TOOLS = editor tsbuild ralint
 
 VERSION     = $(shell git name-rev --name-only --tags --no-undefined HEAD 2>/dev/null || echo git-`git rev-parse --short HEAD`)
-- 
1.8.4
